synthpops/contact_networks_industries.py

```diff
@@ -261,7 +272,21 @@ def get_establishment_size_brackets_df(datadir, locations, state_location='Washi
 
 def generate_synthetic_population_with_workplace_industries(n, datadir,location='seattle_metro',state_location='Washington',country_location='usa',sheet_name='United States of America',level='county',verbose=False,plot=False):
     """
-    
+    Modify the workplace network as generated by :py:meth:`~synthpops.generate_synthetic_population` to include  contact patterns according to each industry.
+
+    Args:
+        n (int)                                   : The number of people in the population.
+        datadir (string)                          : The file path to the data directory.
+        location (string)                         : The name of the location.
+        state_location (string)                   : The name of the state the location is in.
+        country_location (string)                 : The name of the country the location is in.
+        sheet_name (string)                       : The name of the sheet in the Excel file with contact patterns.
+        level (string)                            : The scale of region at which data is available.
+        verbose (bool)                            : If True, print statements as contacts are being generated.
+        plot (bool)                               : If True, plot and show a comparison of the generated workplace sizes vs. the expected sizes based on NAICS data.
+
+    Returns:
+        None
 
```

Comment:
This function and the whole script is not ready yet because it tends to produce incredibly large workplaces due to large enterprises like factories likely reporting the number of workers for multiple shifts. Is there anyway we can leave this script out of the repo until I can find a better solution for this part? Might be able to get something ready this weekend or next week if dev help is happening.